parsers/pascal.c

```diff
@@ -276,11 +278,9 @@ static void findPascalTags (void)
 				continue;
 			vStringNCopyS (name, (const char*) dbp,  cp - dbp);
 
-			if (arglist != NULL)
-				eFree(arglist);
-			if (kind == K_FUNCTION && vartype != NULL)
-				eFree(vartype);
-			parseArglist((const char*) cp, &arglist, (kind == K_FUNCTION) ? &vartype : NULL);
+			vStringClear (arglist);
+			vStringClear (vartype);
+   			parseArglist((const char*) cp, arglist, (kind == K_FUNCTION) ? vartype : NULL);
```

Comment:
A minor issue. The line 282 and the line 283 are not aligned.

Reply:
Line 282 uses only tag characters.
However, the line 283 uses tabs and spaces mixed way.

Reply:
Oops, will fix.

Another thing I noted (and probably was introduced by me in 2009), there are some inconsistencies in using a space between function name and the opening brace of the argument list. Should I fix them as well, to consistently use a space between?

Reply:
> Should I fix them as well, to consistently use a space between?

Thank you for the offering. If possible, could you fix them in ANOTHER commit?
In a commit, let's do one thing.

Reply:
Ok, the wrong indentation is fixed.

Did you mean another commit in this PR to fix the argument list space problem or a separate PR?
I don't mind.

Reply:
I also don't mind. However, you already submitted a good commit that aligns with the goal of this pull request.
So I will merge this.